### PR TITLE
Enable muon trigger matching

### PR DESCRIPTION
**Pull request by ntadej**
`m_doTrigMatch` should be `true` for muon matching to work at all.

### CODE REVIEW

File: xAODAnaHelpers/MuonSelector.h [hunk 1]
```diff
@@ -158,7 +158,7 @@ class MuonSelector : public xAH::Algorithm
   */
   asg::AnaToolHandle<Trig::TrigDecisionTool>       m_trigDecTool_handle{"Trig::TrigDecisionTool"};                  //!
 
-  bool m_doTrigMatch = false; //!
+  bool m_doTrigMatch = true; //!
```
Comment (kratsg) [inline xAODAnaHelpers/MuonSelector.h]: can you add a comment (for documentation) on what this variable does?

Comment (ntadej) [inline xAODAnaHelpers/MuonSelector.h]: sure, I can also add it for electrons for consistency?

Comment (kratsg) [inline xAODAnaHelpers/MuonSelector.h]: yes

**Comment by ntadej** [comment]
Updated with a comment.

File: xAODAnaHelpers/ElectronSelector.h [hunk 2]
```diff
@@ -261,6 +261,8 @@ class ElectronSelector : public xAH::Algorithm
   */
   asg::AnaToolHandle<Trig::TrigDecisionTool>      m_trigDecTool_handle{"Trig::TrigDecisionTool"};                //!
   asg::AnaToolHandle<Trig::IMatchingTool>         m_trigElectronMatchTool_handle{"Trig::MatchingTool"};          //!
+
+  // This internal variable gets set to false if no triggers are defined or if TrigDecisionTool is missing
```
Comment (kratsg) [inline xAODAnaHelpers/ElectronSelector.h]: needs to be `/// @brief ...` for doxygen-style comments.

File: xAODAnaHelpers/MuonSelector.h [hunk 3]
```diff
@@ -158,7 +158,8 @@ class MuonSelector : public xAH::Algorithm
   */
   asg::AnaToolHandle<Trig::TrigDecisionTool>       m_trigDecTool_handle{"Trig::TrigDecisionTool"};                  //!
 
-  bool m_doTrigMatch = false; //!
+  // This internal variable gets set to false if no triggers are defined or if TrigDecisionTool is missing
```
Comment (kratsg) [inline xAODAnaHelpers/MuonSelector.h]: needs to be `/// @brief ....` for doxygen-style comments.

**Pull request merged**